src/GraphWebhooks-Core/GraphWebhooks-Core.csproj

```diff
@@ -32,7 +32,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="2.2.0" />
     <PackageReference Include="Microsoft.AspNetCore.SignalR.Common" Version="1.1.0" />
     <PackageReference Include="Microsoft.AspNetCore.SignalR.Core" Version="1.1.0" />
-    <PackageReference Include="Microsoft.Graph" Version="1.12.0" />
+    <PackageReference Include="Microsoft.Graph" Version="1.14.0" />
```

Comment:
Let's use a package version of 1.* so that this gets run against the latest version of the SDK.

Reply:
Done